Cargo.toml

```diff
@@ -1,14 +1,11 @@
 [workspace]
 members = [
 	"typify",
+	"typify-cli",
 	"typify-macro",
 	"typify-impl",
 	"example-build",
 	"example-macro",
 ]
 
-default-members = [
-	"typify",
-	"typify-macro",
-	"typify-impl",
-]
+default-members = ["typify", "typify-cli", "typify-macro", "typify-impl"]
```

Comment:
is there a reason this is all on one line?

Reply:
'cargo fmt' return this